tests/appsec/test_blocking_addresses.py

```diff
@@ -12,7 +12,7 @@
     python={"django-poc": "1.10", "flask-poc": "1.10", "*": "?"},
     nodejs="3.19.0",
     golang="1.51.0",
-    ruby="1.0.0",
+    ruby="1.12.0",
```

Comment:
If you just missed one feature, you might want to keep `1.0.0` here and change the next `@missing_feature(context.library == "ruby")` to `@missing_feature(context.library < "ruby@1.12.0")`